build_tools/travis/install.sh

```diff
@@ -37,15 +37,20 @@ if [[ "$DISTRIB" == "conda" ]]; then
     echo "Cached in $HOME/download :"
     ls -l
     echo
-    if [[ ! -f miniconda.sh ]]
-        then
-        wget http://repo.continuum.io/miniconda/Miniconda-3.6.0-Linux-x86_64.sh \
-            -O miniconda.sh
+    if [[ ! -f miniconda.sh ]]; then
+        if [[ "$PYTHON_VERSION" == "2.7" ]]; then
```

Comment:
oh duh, it doesn't matter since you're specifying a Python version when creating the env. whoops